TransformationSystem/Agent/InputDataAgent.py

```diff
@@ -91,54 +101,58 @@ def execute( self ):
         if transID in self.timeLog:
           if transID in self.fullTimeLog:
             # If it is more than a day since the last reduced query, make a full query just in case
-            if ( datetime.datetime.utcnow() - self.fullTimeLog[transID] ) < datetime.timedelta( seconds = self.fullUpdatePeriod ):
+            if (
```

Comment:
Remove the "new line"... It looks really ugly like that...

Reply:
Done.